MCprep_addon/materials/generate.py

```diff
@@ -1522,16 +1529,19 @@ def matgen_cycles_principled(
 			# Note: placed with hasattr to reduce bugs, seemingly only on old
 			# 2.80 build
 			if hasattr(mat, "blend_method"):
-				mat.blend_method = 'HASHED'
+				mat.blend_method = util.BLEND_MODE
 			if hasattr(mat, "shadow_method"):
-				mat.shadow_method = 'HASHED'
+				mat.shadow_method = util.SHADOW_MODE
 
 			# best if there is no partial transparency
 			# material.blend_method = 'CLIP' for no partial transparency
 			# both work fine with depth of field.
 
 			# but, BLEND does NOT work well with Depth of Field or layering
 
+			# This prevents transparency issues with Alpha Blend
+			mat.show_transparent_back = False
```

Comment:
Ah, what happens here? Do you have a screenshot of with this off vs on in a relevant context? Just curious.

Reply:
Right now I don't have a screenshot on hand but without disabling this, any materials using alpha transparency (when using Alpha Blend) will be really messed up. This option does not affect Alpha Hashed.

Reply:
Great, good to know it doesn't affect alpha hashed, thanks!